backend/request.ts

```diff
@@ -707,6 +707,7 @@ export async function parseFilterProjectsRequest(
         clientNameFilter: maybe(req.body, "clientNameFilter"),
         assignedCoachesFilterArray: assignedCoachesFilterArray,
         fullyAssignedFilter: fullyAssignedFilter,
+        osocYearFilter: maybe(req.body, "osocYearFilter"),
```

Comment:
I would replace all occurrences of `osocYearFilter` with `osocYear` because that is what the student filter currently uses. So let's make it uniform.